api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -34,8 +34,8 @@
     private GenericSecretSource clientSecret;
     private String validIssuerUri;
     private String jwksEndpointUri;
-    private int jwksRefreshSeconds;
-    private int jwksExpirySeconds;
+    private Integer jwksRefreshSeconds;
+    private Integer jwksExpirySeconds;
```

Comment:
Why do we need this change? `0` is invalid value so I think it can be easily `int`.